src/createDicomWebTreeApi.ts

```diff
@@ -529,6 +533,22 @@ function createDicomWebTreeApi(dicomWebConfig, UserAuthenticationService) {
       });
       return imageIds;
     },
+    getStudyInstanceUIDs({ params, query }: { params: any; query: any}) {
```

Comment:
This method was introduced in the newer version of OHIF and I am adding it here just to be compatible with latest v3